babel.config.js

```diff
@@ -12,6 +12,9 @@ module.exports = {
           '@/core': './standalone/src/core',
           '@/core/test-utils': './standalone/src/core/test-utils',
           '@/graphql-types': './standalone/src/graphql-types',
+          '@/models/cart': './standalone/src/models/cart/cart',
+          '@/models/cart-discount':
+            './standalone/src/models/cart/cart-discount',
```

Comment:
@CarlosCortizasCT FYI: we need to adjust the path aliases as the folder hierarchy changed.

There are different options we can go fo:
1. Explicitly mapping nested models. We would need to adjust these mappings for all nested / grouped models.
2. For the grouped models, define a new proxy entry point (e.g. `./models/cart.ts` and `./models/cart-discounts.ts`) so that we retain the generic mapping of path aliases `"@/models/*": ["./standalone/src/models/*"]`
3. Change the mapping to `"@/models/*": ["./standalone/src/*"]` to rely on the package entry points. However, an import like `@/models/graphql-types` or `@/models/core` is then also possible which makes it weird.

Reply:
I vote against adding new aliases as it adds an extra layer of configuration I think we don't need.

If we regroup directories we just need to update the imports once and it seems that's something that can easily be done with "Find & Replace" IDE's feature.

The PR will be big anyway because we're moving around many files.

Reply:
Thanks, I would also prefer to avoid adding new aliases.

Then, is this way of importing ok with you?

```ts
import { Cart } from '@/models/cart/cart';
import { CartDiscount } from '@/models/cart/cart-discount';
```

Reply:
Internally in this repo, yes it's fine by me.

Reply:
https://github.com/commercetools/test-data/pull/810/commits/05f0f4150fd142986e3fc073abba47e8db243859